Writerside/topics/Accounts.md

```diff
@@ -1,149 +1,83 @@
 # Accounts
 
-The Catena `AccountsService` is responsible for creating, fetching, updating, and deleting accounts and their associated metadata.
+Catena provides an accounts system that is designed for maximum flexiblity. The rest of Catena is designed to not depend on the Account implementation, as a result, you can bring your own account implementation and swap it out for Catena's.
```

Comment:
> not to depend on the Account implementation

I think the wording here is a bit confusing since some things need accounts. Maybe "not to depend on Catena's **provided** account implementation"

Reply:
I like this better.